pkgs/development/python-modules/scrapy/default.nix

```diff
@@ -93,5 +93,6 @@ buildPythonPackage rec {
     license = licenses.bsd3;
     maintainers = with maintainers; [ drewkett marsam ];
     platforms = platforms.unix;
+    broken = true;
```

Comment:
scrapy is not broken:
- x86_64-linux: https://hydra.nixos.org/build/116062539
- x86_64-darwin: https://hydra.nixos.org/build/116118595

Reply:
Yeah. Looks like a pattern. Same as in https://github.com/NixOS/nixpkgs/pull/85417/files#r410125598

Reply:
Is it broken on release-20.03 though?

Reply:
I usually look whether something is broken by editing these urls:
https://hydra.nixos.org/jobset/nixos/release-20.03/jobs-tab?filter=scrapy
https://hydra.nixos.org/jobset/nixpkgs/trunk/jobs-tab?filter=scrapy